astropy/visualization/interval.py

```diff
@@ -6,39 +6,73 @@
 """
 
 from __future__ import division, print_function
-
```

Comment:
I'm not sure if it's really recommended. [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) states they should be grouped and I always interpreted this as blank lines. :smile: